src/microbit/test/test_display.py

```diff
@@ -0,0 +1,148 @@
+import pytest
+import threading
+from unittest import mock
+
+from ..model import constants as CONSTANTS
+from ..model.display import Display
+from ..model.image import Image
+from .. import shim
+
+
+STR_A = "09900:90090:99990:90090:90090"
+STR_QUESTION_MARK = "09990:90009:00990:00000:00900"
+STR_EXCLAMATION_MARK = "09000:09000:09000:00000:09000:"
+STR_SIX = "00090:00900:09990:90009:09990"
+
+
+class TestDisplay(object):
+    def setup_method(self):
+        self.display = Display()
+
+    @pytest.mark.parametrize("x, y, brightness", [(1, 1, 4), (2, 3, 6), (4, 4, 9)])
+    def test_get_pixel(self, x, y, brightness):
+        self.display._Display__image._Image__LED[y][x] = brightness
+        assert brightness == self.display.get_pixel(x, y)

```

Comment:
I would merge `test_get_pixel` and `test_set_pixel` into a single test and only test `display.set_pixel`/`display.get_pixel` instead of ` _Display__image._Image__LED`.
In general we want to focus on testing behavior, rather than implementation. If we change the implementation, ideally we don't want to have tests failing.

Reply:
Agreed, changed